colorbleed/scripts/fusion_switch_shot.py

```diff
@@ -33,7 +33,7 @@ def _format_version_folder(folder):
     if not os.path.isdir(folder):
         return "v001"
 
-    re_version = re.compile("v\d+")
+    re_version = re.compile("v\d+$")
 
     content = sorted(os.listdir(folder))
```

Comment:
Not to be a bitch, note that - in this specific scenario - there's no reason to actually sort the output of `os.listdir`. Not really an issue though. :smile:

Reply:
Oh darn that's true :) probably as a result of assuming there would be only v### folder but I changed my mind about that.